### PR TITLE
Fix can't delete order items

### PR DESCRIPTION
**Pull request by lumosnysm**

### CODE REVIEW

**Comment by lumosnysm** [comment]
ready

File: app/controllers/order_items_controller.rb [hunk 1]
```diff
@@ -50,4 +48,9 @@ def destroy
   def order_item_params
     params.require(:order_item).permit(:quantity, :dish_id, :price_each)
   end
+
+  def load_order_item
+    @order = current_order
+    @order_item = @order.order_items.find_by id: params[:id]
```
Comment (mdmahmudurrahman) [inline app/controllers/order_items_controller.rb]: Em, check nil.

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by lumosnysm** [comment]
ready

**Pull request merged**